nutils/points.py

```diff
@@ -301,27 +300,27 @@ def masks(self):
     for pairs in self.duplicates:
       for i, j in pairs[1:]:
         masks[i][j] = False
-    return tuple(types.frozenarray(m, copy=False) for m in masks)
+    return types.tuple[numeric.freeze](masks)
```

Comment:
I'm not sure about using `types.tuple[f](s)` here either. It's hardly shorter than `tuple(map(f, s))`

Reply:
fixed